src/server/index.ts

```diff
@@ -223,6 +223,12 @@ initDb()
         res.badRequest(jsonMessage(err.error.toString()))
         return
       }
+
+      if (err.status === 400) {
+        res.badRequest(jsonMessage("Bad Request. JSON is malformed"))
+        return;
+      }
```

Comment:
hey @developerdenesh, thanks for contributing! this works well, but a possible concern is that errors with status 400 that are not necessarily JSON parsing errors will be caught and handled under this block as well.

perhaps we could strengthen the if condition to narrow it down to JSON errors? I've just checked and found an answer [here](https://stackoverflow.com/a/53049009) that tries to do just that, as well as another possible candidate `type: 'entity.parse.failed'` described [here](https://expressjs.com/en/resources/middleware/body-parser.html#errors) that we may want to check on as well - hope it helps!

Reply:
That's true. Have made the changes in #2042